site/src/pages/tutorials/tutorial-blog-rest-step3.mdx

```diff
@@ -0,0 +1,169 @@
+    ```java filename=BlogRequestConverter.java highlight=9,10,13
+    @Override
+    public Object convertRequest(ServiceRequestContext ctx,
+      AggregatedHttpRequest request, Class<?> expectedResultType,
+      @Nullable ParameterizedType expectedParameterizedResultType)
+        throws Exception {
+      if (expectedResultType == BlogPost.class) {
+        final JsonNode jsonNode = mapper.readTree(request.contentUtf8());
+        final int id = idGenerator.getAndIncrement();
+        final String title = stringValue(jsonNode, "title");
+        final String content = stringValue(jsonNode, "content");
+        return new BlogPost(id, title, content); // Create an instance of BlogPost object
+      }
+      return RequestConverterFunction.fallthrough();
+    }
+  ```
+
+### Assign a request converter
+
+In this step, assign the [request converter we customized](#write-a-request-converter) to our service method. Annotate the service method with <type://@RequestConverter> and specify the RequestConverterFunction class as `BlogPostRequestConverter.class`.
+
+```java filename=BlogService.java highlight=2
+@Post("/blogs")
+@RequestConverter(BlogPostRequestConverter.class)
+public HttpResponse createBlogPost(BlogPost blogPost) throws JsonProcessingException {
+  // Implement blog service
+}
+```
+
+## 3. Implement service code
+
+When the request for creation is received, our request converter creates an instance of a blog post object for us. We want to save the blog post object in the map (`blogPosts`) created in the `BlogService` class. Skip adding a database operation for this tutorial.
+
+To store the blog post information in the map, add the following line in the `createBlogPost()` method.
+
+```java filename=BlogService.java highlight=2
+public HttpResponse createBlogPost(BlogPost blogPost) throws JsonProcessingException {
+  blogPosts.put(blogPost.id(), blogPost);
+}
+```
+
+## 4. Return response
+
+Now, it's time to return a response to our client. As the response, on top of the information received, add the ID of the post, created time, plus modified time which would be identical to the created time.
+
+1. Specify JSON object keys for each member of our blog post object, by annotating the following methods with `@JsonProperty`.
+  ```java filename=BlogPost.java highlight=3,6,9,12,15
+  final class BlogPost {
+    ...
+    @JsonProperty("id")
+    int id() { return id; }
+
+    @JsonProperty("title")
+    String title() { return title; }
+
+    @JsonProperty("content")
+    String content() { return content; }
+
+    @JsonProperty("createdAt")
+    long createdAt() { return createdAt; }
+
+    @JsonProperty("modifiedAt")
+    long modifiedAt() { return modifiedAt; }
+    ...
+  }
+  ```
+2. Create and return an HTTP response using Armeria's <type://HttpResponse> and include the information of the post created, as shown below, in the `createBlogPost()` method.
```

Comment:
In my humble opinion, we can remove `as shown below` throughout the tutorials.

Reply:
All "as shown below"s are gone with the wind :)